gensim/test/test_atmodel.py

```diff
@@ -450,6 +450,31 @@ def testTermTopics(self):
             self.assertTrue(isinstance(topic_no, int))
             self.assertTrue(isinstance(probability, float))
 
+    def testNewAuthorTopics(self):
```

Comment:
CC: @olavurmortensen, this test looks like "sanity-check" only, can you propose something more significant?

Reply:
Besides the sanity check, I tried another test case, where I would infer the topics for a new author, using the exact same documents from the corpus that an author already is assigned to. We can expect that the topic distributions should be very similar, as it is, but not close enough so that we can use np.allclose(..,..). Example:

In [2]: model.get_new_author_topics(corpus=corpus[1:3])
Out[2]: [(0, 0.914887958236441), (1, 0.08511204176355897)]

In [3]: model.get_author_topics(author_name="sally")
Out[3]: [(0, 0.9290420023752036), (1, 0.07095799762479651)]

Reply:
Sounds reasonable, only one thing - may be better to calculate [JS-divergense](https://en.wikipedia.org/wiki/Jensen%E2%80%93Shannon_divergence) instead of `allclose` with very hight `atol`?

Reply:
https://github.com/RaRe-Technologies/gensim/blob/a54e3369bf832bc632d20f5e1d401267e6bc4882/gensim/matutils.py#L921 - implementation already available in gensim

Reply:
How would you propose to set the similarity threshold? Just empirically for this one test case?

Reply:
yes, I don't know the better solution here.